src/cpp/src/continuous_batching_impl.cpp

```diff
@@ -297,11 +296,7 @@ ContinuousBatchingPipeline::ContinuousBatchingImpl::generate(const std::vector<o
         streamer_ptr->end();
     }
 
-    if (!continue_generation) {
-        drop_requests();
```

Comment:
why have you removed `drop_requests` here ? Looks like `continue_generation` should be assigned to `!streamer_ptr->put` return value?

Looks like the logic was broken in https://github.com/openvinotoolkit/openvino.genai/pull/1377